src/patternfly/components/Page/page.scss

```diff
@@ -82,24 +82,13 @@
   }
 
   // Main section
-  --pf-c-page__main-section--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-page__main-section--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-page__main-section--PaddingBottom: var(--pf-global--spacer--md);
-  --pf-c-page__main-section--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-page__main-section--md--PaddingTop: var(--pf-global--spacer--xl);
-  --pf-c-page__main-section--md--PaddingRight: var(--pf-global--spacer--xl);
-  --pf-c-page__main-section--md--PaddingBottom: var(--pf-global--spacer--xl);
-  --pf-c-page__main-section--md--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-page__main-section--PaddingTop: var(--pf-global--spacer--xl);
```

Comment:
Just want to verify that this is purposeful. The spacing in other elements in the page component change at a medium breakpoint, so the content no longer aligns properly. 

This is how it was before:

<img width="722" alt="screen shot 2019-02-19 at 4 11 43 pm" src="https://user-images.githubusercontent.com/35148959/53051290-17f64b80-3461-11e9-92b2-9111a4ca42e8.png">

And here's what happens now, including the code below that removes the padding on the last section area on a mobile breakpoint

<img width="722" alt="screen shot 2019-02-19 at 4 11 49 pm" src="https://user-images.githubusercontent.com/35148959/53051326-2fcdcf80-3461-11e9-9203-6c964a33cf0b.png">

Reply:
This is a really good point. We want this styling for the table examples where the `.pf-c-page__main-section` has no padding when the page responds. But I wonder if we want it for every example for the page on mobile - and I think not. I wonder if we should create a modifier to add to `.pf-c-page__main-section` that has no padding on mobile so that it can hold a table, or other examples that dont need padding in the responsive state. What do you think? @mcoker

Reply:
Right, I don't think we want to change the default page layout just to accommodate these examples or use cases. FWIW, it still looks like you've removed the variable main-section padding between small/md viewports, and are using the desktop padding on both breakpoints except the bottom padding of the main section is larger at both breakpoints now, too.

We used to use `md` padding all around on mobile, and `xl` padding on desktop. Now we're using `xl` for top, left, and right, and `2xl` for bottom padding on both mobile and desktop. I just want to verify that this fundamental change to the page layout spacing is part of this PR. If so, this also implies that we need to make global spacing changes to the masthead and navigation in a mobile viewport to match this new spacing.

This is how the masthead, navigation, and content align on the left edge currently.

<img width="783" alt="screen shot 2019-02-21 at 5 06 25 pm" src="https://user-images.githubusercontent.com/35148959/53208043-119bd700-35fb-11e9-876f-4e1eddfe3596.png">

<img width="783" alt="screen shot 2019-02-21 at 5 06 28 pm" src="https://user-images.githubusercontent.com/35148959/53208051-16f92180-35fb-11e9-8c90-a85d3fc1b704.png">

This is how the content now aligns with this change. If we keep this main section padding as it is, I'm assuming we'll need to update the masthead left/right padding so those components align on the left edge, and since the masthead padding changed, the navigation padding will need to change, so that when it's open, it now aligns with the new padding in the masthead.

<img width="783" alt="screen shot 2019-02-21 at 5 07 11 pm" src="https://user-images.githubusercontent.com/35148959/53208077-29735b00-35fb-11e9-8ff4-7e2d56fef9a3.png">

Reply:
re: the modifier for an alternate spacing on the main section, I assume it's not an option just to keep the spacing that currently exists? (16px or `--pf-global--spacer--xl` all around).  And we've talked recently about putting together demos that have custom styling or use a utility for spacing as a demonstration of _something a user can do_, instead of _something you get by default from patternfly_, specifically around building card demos. Just want to verify this is something we want to ship with? If so I could see the use for a simple modifier that just removes the padding entirely or maybe removing the default spacing and adding an element that creates the spacing (like we do in `Card` and are doing in `Wizard`) and just leave the area up to the user to handle their own spacing, but that wouldn't work for the examples in this PR because of the small `padding-top` applied. 

Assuming this change should ship with patternfly, it's fine as is, but I doubt this spacing will just be used for a table (or maybe it will?), so I think a more meaningful class would be better. I'm not sure what that name would be, so maybe we just leave it :)

Reply:
hey @christiemolloy just checking to see if you saw my comment above? I'm assuming this would also impact https://github.com/patternfly/patternfly-next/pull/1401

Reply:
@mcoker Based on the designs, this padding around the card is something that we would want to ship with Patternfly it shoudn't be custom because it is the default styling for cards that have data tables/data lists, on mobile. IMO we haven't had enough examples on cards to see all the variations for page mobile styling. As of right now, I think we should have the standard padding and then a modifier for this padding but I'm going to check with design!